service/operator/operatorkit.go

```diff
@@ -0,0 +1,146 @@
+package operator
+
+import (
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+// Operator implements the building blocks of any operator business logic being
+// reconciled when observing TPRs. This interface provides a guideline for a
+// more easy way to follow the rather complex intentions of operators in
```

Comment:
I think it's wrong, but I'm no native speaker. https://english.stackexchange.com/questions/67991/much-more-easy-versus-much-easier#67993

@JosephSalisbury please help us.